helm/dns-operator-aws/values.yaml

```diff
@@ -16,7 +16,7 @@ managementClusterName: ""
 managementClusterNamespace: ""
 
 
-associateResolveRules: false
+associateResolverRules: false
```

Comment:
dont we need to change some config repo as well? and maybe mc-bootstrap?

Reply:
https://github.com/giantswarm/config/blob/main/installations/thunder/apps/dns-operator-aws/configmap-values.yaml.patch

Reply:
https://github.com/giantswarm/mc-bootstrap/blob/update-proxy-stack/scripts/setup-config-branch.sh#L242

Reply:
What's missing so we can merge mc-bootstrap changes into main? It's hard to keep track of what's on every branch on different repositories.

Reply:
Param change on mc-bootstrap https://github.com/giantswarm/mc-bootstrap/pull/366

Reply:
I think the main reason we have this stale branch is that we wanted stable mc-bootstrap in order to be able to create talanx  MCs, as rocket is doing lot of changes for the CI and there was an agreement to create a branch that we know works for us.

Reply:
Param change on the config repo https://github.com/giantswarm/config/pull/1465
We can't merge this PR until this one is merged and released.

Reply:
`thunder` ahs anyway stopped all flux and updates so no merge of PR will update anything there